File: DEMObarcodeScanner/ViewController.m
//
//  ViewController.m
//  DEMObarcodeScanner
//
//  Created by Dipen Lad on 12/2/16.
//  Copyright © 2016 cstechsolutions. All rights reserved.
//

#import "ViewController.h"
#import "MTBBarcodeScanner.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    MTBBarcodeScanner *scanner = [[MTBBarcodeScanner alloc] initWithPreviewView:self.viewScanner];
    
    [MTBBarcodeScanner requestCameraPermissionWithSuccess:^(BOOL success) {
        if (success) {
            
            NSError *error = nil;
            [scanner startScanningWithResultBlock:^(NSArray *codes) {
                AVMetadataMachineReadableCodeObject *code = [codes firstObject];
                NSLog(@"Found code: %@", code.stringValue);
                
                [scanner stopScanning];
            } error:&error];
            
        } else {
            // The user denied access to the camera
        }
    }];
    
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: SwiftDemo/ViewController.swift
//
//  ViewController.swift
//  SwiftDemo
//
//  Created by Dipen Lad on 8/22/16.
//  Copyright © 2016 Dipen Lad. All rights reserved.
//

import UIKit
import Alamofire
import SwiftyJSON

class ViewController: UIViewController, UITextFieldDelegate
{
    @IBOutlet weak var txtUserName : UITextField!
    @IBOutlet weak var lblValue : UILabel!
    @IBOutlet weak var btnSend : UIButton!
    @IBOutlet weak var btnNext : UIButton!

    override func viewDidLoad()
    {
        super.viewDidLoad()
        
        self.lblValue.text = MySingleton.sharedInstance.strValue as String;
        
        
        Alamofire.request(.GET, "http://api.androidhive.info/contacts/").response { (req, res, data, error) -> Void in
            print(res)
            let outputString = NSString(data: data!, encoding:NSUTF8StringEncoding)
            print(outputString)
        }
        
        btnSend.addTarget(self, action: #selector(self.btnSendClicked(_:)), forControlEvents:  .TouchUpInside);
        btnNext.addTarget(self, action: #selector(self.btnNextClicked(_:)), forControlEvents:  .TouchUpInside);
        
        // Do any additional setup after loading the view, typically from a nib.
    }
    
    @IBAction func btnSendClicked(sender : AnyObject)
    {
        
        txtUserName.resignFirstResponder();
        lblValue.text = txtUserName.text;
    }
    
    @IBAction func btnNextClicked(sender :AnyObject)
    {
        let NextVC = secondViewController(nibName: "secondViewController",bundle: nil)
        self.navigationController?.pushViewController(NextVC, animated: true)
        
    }
    

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

